release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,91 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id=ocp-4-11-monitoring-updates-to-monitoring-stack-components-and-dependencies]
+==== Updates to monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.5.0
+* Prometheus to 2.36.2
+* Prometheus operator to 0.57.0
+* Thanos to 0.26.0
+
+[id=ocp-4-11-monitoring-changes-to-alerting-rules]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `KubePersistentVolumeInodesFillingUp` alert, which functions similarly to the existing `KubePersistentVolumeFillingUp` alert but is applied to inodes rather than volume space.
+** Added the `PrometheusScrapeBodySizeLimitHit` alert to detect targets hitting the body size limit.
+** Added the `PrometheusScrapeSampleLimitHit` alert to detect targets hitting the sample limit.
+
+* *Changed*
+** Fixed the `KubeDaemonSetRolloutStuck` alert to use the updated metric `kube_daemonset_status_updated_number_scheduled` from `kube-state-metrics`.
+** Replaced the `KubeJobCompletion` alert with `KubeJobNotCompleted`. The new `KubeJobNotCompleted` alert avoids false positives when an earlier job failed but the most recent job succeeded.
+** Updated the `NodeNetworkInterfaceFlapping` alert to exclude `tunbr` interfaces from the alert expression.
+
+[id=ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring]
+==== Enable alert routing for user workload monitoring
+A cluster administrator can now enable alert routing for user workload monitoring so that developers and other users can configure custom alerts and alert routing for their user-defined projects.
+
+[id=ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts]
+==== Enable a dedicated Alertmanager instance for user-defined alerts
+You can optionally enable a separate instance of Alertmanager dedicated to sending alerts only for user-defined projects.
```

Comment:
Word choice

I think `You have the option to enable a separate instance...` reads better and would make translation easier. 

`optionally` is a bit of a weird adverb in my opinion. However, the original sentence reads perfectly well, so feel free to leave as is :)

Reply:
Your edit works better. Will make the change.